### PR TITLE
Adding a pattern to copy an object between two S3 buckets using S3 Event Notification to SQS to EventBridge Pipes to Step Function

### PR DESCRIPTION
**Pull request by dastra**
*Issue #, if available:*  #2086

*Description of changes:*

This sample project demonstrates how to copy an object between two S3 buckets and manipulate it in transit. When an object is created in the source S3 bucket, an S3 Event Notification is fires and enqueued onto an SQS Queue. The EventBridge service polls the SQS Queue and invokes an EventBridge pipe. The EventBridge Pipe then executes an AWS Step Functions Express Workflow state machine. This copies the file from the source S3 Bucket to the destination S3 Bucket. In a real world scenario the state machine would be modified to manipulate the file as desired

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by parikhudit** [comment]
@dastra Thanks for your contribution. Would you like to add a note that this would work for JSON type objects only or modify the code to support all media types? I think SFN may fail trying to convert non-JSON media types to string.

File: s3-sqs-eventbridge-pipe-sfn-s3/s3-sqs-eventbridge-pipe-sfn-s3.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+  "title": "Copy an object between two S3 buckets using S3 Event Notification to SQS to EventBridge Pipes to Step Function",
```
Comment (parikhudit) [inline s3-sqs-eventbridge-pipe-sfn-s3/s3-sqs-eventbridge-pipe-sfn-s3.json:2]: Length needs to be below 75 characters

File: s3-sqs-eventbridge-pipe-sfn-s3/s3-sqs-eventbridge-pipe-sfn-s3.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  "description": "Create a Step Functions workflow to query Amazon Athena.",
```
Comment (parikhudit) [inline s3-sqs-eventbridge-pipe-sfn-s3/s3-sqs-eventbridge-pipe-sfn-s3.json:3]: Could you kindly update this?

File: s3-sqs-eventbridge-pipe-sfn-s3/s3-sqs-eventbridge-pipe-sfn-s3.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    "text": [
+      "Delete the stack: <code>sam delete</code>."
+    ]
+  },
+  "authors": [
+    {
+      "name": "Dan Straw",
+      "image": "https://avatars.githubusercontent.com/u/271028",
+      "bio": "Senior Solutions Architect at AWS",
+      "linkedin": "https://www.linkedin.com/in/danstraw/"
```
Comment (parikhudit) [inline s3-sqs-eventbridge-pipe-sfn-s3/s3-sqs-eventbridge-pipe-sfn-s3.json:58]: only LinkedIn id is required e.g. danstraw

File: s3-sqs-eventbridge-pipe-sfn-s3/s3-sqs-eventbridge-pipe-sfn-s3.json (hunk 1 above)
Comment (dastra) [reply s3-sqs-eventbridge-pipe-sfn-s3/s3-sqs-eventbridge-pipe-sfn-s3.json]: Updated!

File: s3-sqs-eventbridge-pipe-sfn-s3/s3-sqs-eventbridge-pipe-sfn-s3.json (hunk 2 above)
Comment (dastra) [reply s3-sqs-eventbridge-pipe-sfn-s3/s3-sqs-eventbridge-pipe-sfn-s3.json]: Updated!

File: s3-sqs-eventbridge-pipe-sfn-s3/s3-sqs-eventbridge-pipe-sfn-s3.json (hunk 3 above)
Comment (dastra) [reply s3-sqs-eventbridge-pipe-sfn-s3/s3-sqs-eventbridge-pipe-sfn-s3.json]: Updated!

**Pull request merged**